docs/pages/upgrading/overview.mdx

```diff
@@ -33,29 +33,29 @@ Teleport cluster:
    migrations on its backend. Follow the instructions in [Backup and
    Restore](../admin-guides/management/operations/backup-restore.mdx).
 
-1. If several Auth Service instances are running in a high availability
-   configuration (for example, in an AWS Auto Scaling group), you must shrink
-   the group to **just one Auth Service** before performing an upgrade.
-
-1. Upgrade the **Auth Service** to the next **major version first**. If there
-   are data format changes introduced in the new version, the Auth Service
-   performs the necessary migrations.  After the upgrade, start the Auth Service
-   and CONFIRM that it's in a healthy state before continuing.
+1. Upgrade all **Auth Service** instances to the **target version first**.
+   Auth Service instances may be upgraded in any sequence or at the same time. After
+   the upgrade CONFIRM that the cluster is in a healthy state before continuing.
```

Comment:
The all-caps CONFIRM makes it seem it's some official term that we defined somewhere. Should we recommend something specific to check?